experiment-setup/execution/host_sequence/scenario_api.py

```diff
@@ -106,9 +108,6 @@ def sequential_interval_scenario(service, start, iter):
 
 def run_api_seq_scenario(service, start):
     data = [sequential_interval_scenario(service, start, 0)]
```

Comment:
We won't need the return in a list just call `data = sequential_interval_scenario(service, start, 0)`